code/modules/reagents/reagent_containers/glass/bottle.dm

```diff
@@ -519,6 +519,16 @@ var/datum/disease2/disease/wizarditis = null
 	..()
 	reagents.add_reagent(ADMINORDRAZINE, 30)
 
+/obj/item/weapon/reagent_containers/glass/bottle/procizine
+	name = "Procizine Bottle"
+	desc = "A small bottle. Contains a liquid with effects decided on the whim of the gods."
+	icon = 'icons/obj/drinks.dmi'
+	//icon_state = "holyflask"
```

Comment:
delete this

Reply:
hello?

Reply:
Hi? The commented out line. Why would you commit that?

Reply:
Not sure, but other bottles in file have it. Could easily remove them all, see below for similar concerns if I should handle renaming like that.

Reply:
It would be fine by me if you wanted to clean up the rest, but it would be best to do it in a different PR or at the very least a different commit (but then you'd have to squash the existing ones yourself so that we can merge without squashing them)

Reply:
Cleaned up the rest

Reply:
didn't squash the previous commits though

Reply:
Decided not to do the unatomic parts